### ISSUE TITLE
Questions about the IOU loss

### ISSUE THREAD

**Issue by billyceline** [xuannianz/keras-fcos#7]
Hi, thanks for your sharing, I notice that your IOU loss function uses 
losses = tf.reduce_sum(losses * y_centerness_true) / (tf.reduce_sum(y_centerness_true) + 1e-8)
I am a little confused that why you perfomr iou loss times centerness  as the regression loss?

**Comment by billyceline** [comment]
It seems that you used the formula mentioned in this paper: Soft Anchor-Point Object Detection
![image](https://user-images.githubusercontent.com/7619955/71146692-2b309e00-2261-11ea-936f-aa2a5b122f8a.png)
